inst/test_bugs.tst

```diff
@@ -223,3 +223,14 @@
 %! % FIXME: not same as an freshly created 5 x 6.
 %! C = sym('B', [5 6]);
 %! assert (isequal(B, C))
+
+%!test
+%! f1 = fileread ('private/magic_double_str.m');
+%! f2 = fileread ('@sym/private/magic_double_str.m');
+%! assert (strcmp (f1, f2))
```

Comment:
Do we need the exact same files at 2 places? I mean what if something changes at one place, is it necessary to change it at another place too?